zenoh-ext/tests/liveliness.rs

```diff
@@ -179,12 +185,20 @@ async fn test_liveliness_querying_subscriber_brokered() {
     assert_eq!(sample.kind(), SampleKind::Put);
     assert_eq!(sample.key_expr().as_str(), LIVELINESS_KEYEXPR_2);
 
-    drop(token1);
+    assert!(token1.undeclare().await.is_ok());
```

Comment:
Doing an `unwrap()` is preferable in this case since the error will be printed out.
If the test fails in the CI then it's hard to debug what was the error that caused the failure.
Same comment applies to the rest of `assert!(x.is_ok())`